src/astar/rocstar.js

```diff
@@ -199,6 +201,12 @@ const main = async () => {
 
     console.log('Task has been executed!');
 
+    // Calculating balance delta to show fee cost
+    const endProxyBalance = await shibuyaHelper.getBalance(proxyOnParachain);
+    const proxyBalanceDelta = (new BN(proxyBalance.free)).sub(new BN(endProxyBalance.free));
+
+    console.log(`\nAfter execution, Proxy’s balance is ${chalkPipe('green')(new BN(endProxyBalance.free).div(decimalBN).toString())} ${symbol}. The delta of proxy balance, or the XCM fee cost is ${chalkPipe('green')(proxyBalanceDelta.div(decimalBN).toString())} ${symbol}.`);
```

Comment:
Looks like there's some issue with this line. I'm seeing when I run this code.
```
After execution, Proxy’s balance is 1000 SBY. The delta of proxy balance, or the XCM fee cost is -1000 SBY.
```

Reply:
Good call. That is due to balance not re-read after a top-up. I fixed it with the below two commits.

Result:
```
Task has been executed! Waiting for 20 seconds before reading proxy balance.

After execution, Proxy’s balance is 999.938 SBY. The delta of proxy balance, or the XCM fee cost is 0.0619 SBY.
```